src/main/java/seedu/souschef/logic/parser/AppContentParser.java

```diff
@@ -36,6 +37,8 @@ public Command parseCommand(ModelSet modelSet, String userInput, CommandHistory
             return new UniversalParser().parseCommand(history, userInput);
         } else if (context == null || context.equals("Recipe")) {
             return new RecipeParser().parseCommand(modelSet.getRecipeModel(), userInput);
+        } else if (context.equals("HealthPlan")) {
```

Comment:
Please change parameter to "Health Plan"(add space in between!). Because I am thinking of using it as a keyword for messages in generic command classes, I want to make it naturally fit in a sentence.